client/my-sites/plugins-wpcom/jetpack-plugins-panel.jsx

```diff
@@ -160,7 +177,28 @@ class JetpackPluginsPanel extends Component {
 					/>
 				</SectionNav>
 
-				<SectionHeader label={ translate( 'Plugins' ) } />
+				<SectionHeader label={ translate( 'Plugins' ) }>
+					{ config.isEnabled( 'automated-transfer' ) &&
+						<ButtonGroup key="plugin-list-header__buttons-browser">
+							<Button
+								compact
+								href={ browserUrl }
+								onMouseEnter={ this.showPluginTooltip }
+								onMouseLeave={ this.hidePluginTooltip }
+								ref="addPluginButton"
+								aria-label={ translate( 'Browse all plugins', { context: 'button label' } ) }>
+								<Gridicon key="plus-icon" icon="plus-small" size={ 18 } />
+								<Gridicon key="plugins-icon" icon="plugins" size={ 18 } />
+								<Tooltip
+									isVisible={ this.state.addPluginTooltip }
+									context={ this.refs && this.refs.addPluginButton }
+									position="bottom">
+									{ translate( 'Browse all plugins', { context: 'button tooltip' } ) }
```

Comment:
I'm not sure if we need a context here. It's pretty obvious what needs to be translated here.